### ISSUE TITLE
Support of reading spatial files : KML (Keyhole Markup Language)

### ISSUE THREAD

**Issue by simonaubertbd** [amphi-ai/amphi-etl#126]
Hello,

KML is a very common file type for geographical analysis and is now a standard of the Open Geospatial Consortium. Would be could to be able to read it and use the feature of #120  to pursue the analysis,

Best regards,

Simon

**Comment by tgourdel** [comment]
Thanks Simon,
Referencing this for the future as it's not a file type natively supported by geopandas but still possible:
https://medium.com/@nouhailadraidar1/a-beginners-guide-to-handling-geospatial-data-using-python-79c20106167e

**Comment by simonaubertbd** [comment]
As a reference : seems to be at least partially possible
https://github.com/geopandas/geopandas/issues/1552

**Comment by tgourdel** [comment]
Yes, it's possible but it needs to be activated and requires additional dependencies :)

**Comment by martinfleis** [comment]
That's is no longer true. With GeoPandas 1.0 and pyogrio IO engine, you should be able to read most of KML files out of the box.

**Comment by tgourdel** [comment]
Great news, thanks for letting us know @martinfleis!
